### PR TITLE
Update editor feed selection

### PR DESCRIPTION
**Pull request by djabarovgeorge**

### CODE REVIEW

File: apps/web/src/components/templates/in-app-editor/FeedItems.tsx [hunk 1]
```diff
@@ -0,0 +1,237 @@
+import React, { useState } from 'react';
+import { Popover, useMantineTheme, Grid, ColorScheme } from '@mantine/core';
+import { FeedChip } from './FeedChip';
+import { colors, shadows, Text, Tooltip, Button } from '../../../design-system';
+import { Copy, Trash } from '../../../design-system/icons';
+import { useClipboard } from '@mantine/hooks';
+import { FeedEntity } from '@novu/dal';
```
Comment (scopsy) [inline apps/web/src/components/templates/in-app-editor/FeedItems.tsx:7]: @novu/dal import alert :P

File: apps/web/src/components/templates/in-app-editor/FeedItems.tsx [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+function PopoverActionBlock({
+  setOpened,
+  showFeed,
+  feedItem,
+}: {
+  setOpened: (boolean) => void;
+  showFeed: boolean;
+  feedItem?: FeedEntity;
+}) {
+  const colorScheme = useMantineTheme().colorScheme;
```
Comment (scopsy) [inline apps/web/src/components/templates/in-app-editor/FeedItems.tsx:110]:
```suggestion
  const { colorScheme } = useMantineTheme();
```

File: apps/web/src/components/templates/in-app-editor/FeedItems.tsx [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+      <CopyBlock showFeed={showFeed} feedItem={feedItem} />
+      <DeleteBlock setOpened={setOpened} showFeed={showFeed} feedItem={feedItem} />
+    </ActionBlockWrapper>
+  );
+}
+
+function CopyBlock({ showFeed, feedItem }: { showFeed: boolean; feedItem?: FeedEntity }) {
+  const [opened, setOpened] = useState(false);
+
+  const colorScheme = useMantineTheme().colorScheme;
```
Comment (scopsy) [inline apps/web/src/components/templates/in-app-editor/FeedItems.tsx:123]:
```suggestion
  const { colorScheme } = useMantineTheme();
```

I think it's nicer like this wdyt?

File: apps/web/src/components/templates/in-app-editor/TemplateInAppEditor.tsx [hunk 4]
```diff
@@ -1,25 +1,21 @@
+import { useInputState } from '@mantine/hooks';
+import { FeedEntity } from '@novu/dal';
```
Comment (scopsy) [inline apps/web/src/components/templates/in-app-editor/TemplateInAppEditor.tsx:2]: dal import here again :) I've added the linter on master already for us

**Review by scopsy — APPROVED** [review]
This looks great, the main problem is with the @novu/dal imports here. I've added the lint rule on main already so We can merge main later to the root branch of this one.

File: apps/web/src/components/templates/in-app-editor/FeedItems.tsx (hunk 1 above)
Comment (djabarovgeorge) [reply apps/web/src/components/templates/in-app-editor/FeedItems.tsx:7]: Will remove it shortly! thanks for the heads up

File: apps/web/src/components/templates/in-app-editor/TemplateInAppEditor.tsx (hunk 4 above)
Comment (djabarovgeorge) [reply apps/web/src/components/templates/in-app-editor/TemplateInAppEditor.tsx:2]: Yea the linter will really help, so really thanks for it!!!!

**Pull request merged**